src/loader/bgw_message_queue.c

```diff
@@ -155,7 +155,7 @@ queue_reset_reader(MessageQueue *queue)
 				(errcode(ERRCODE_INTERNAL_ERROR),
```

Comment:
Thats kinda odd to ask user to provide a bug report only in one place. I wonder if it makes to make a function out of it and reuse it in different places.

Also I am thinking maybe it is possible to hook into PostgreSQL event handler in order to catch `SIGSEGV` and maybe automatically show or collect stack trace (maybe event send it to us, since we have telemetry already) WDYT?